internal/private/private.go

```diff
@@ -170,6 +171,9 @@ type CommitOptions struct {
 	// What “resolved” means is transport-specific.
 	// Transports which don’t support reporting resolved references can ignore the field; the generic copy code writes "nil" into the value.
 	ReportResolvedReference *types.ImageReference
+	// SourceDateEpoch, if set, and if applicable for destination, will override any creation/modified/access timestamps
+	// useful for oci-archive outputs
+	SourceDateEpoch *time.Time
```

Comment:
(This is internal, so I’m not nearly as worried about the name or about word-smithing the comment. Maybe just `Timestamp` (with the purpose implied by the `CommitOptions` context + copy the comment from `copy.Options`?)

Reply:
Done.